src/snowcli/connection_config.py

```diff
@@ -1,37 +1,30 @@
 from __future__ import annotations
 
 import os
-from configparser import ConfigParser
-from pathlib import Path
+
+from snowflake.connector.config_manager import CONFIG_PARSER
+
 from snowcli.exception import EnvironmentVariableNotFoundError
 
 
 class ConnectionConfigs:
-    def __init__(self, snowsql_config_path="~/.snowsql/config"):
-        self.snowsql_config_path = snowsql_config_path
-        self._config = ConfigParser(inline_comment_prefixes="#", interpolation=None)
-        p = Path(snowsql_config_path).absolute().expanduser()
-        self._config.read(p)
+    def __init__(self):
+        self._config = CONFIG_PARSER
+        self._config.read_config()
 
     def connection_exists(self, connection_name: str):
         return self._config.sections().__contains__(f"connections.{connection_name}")
 
     def get_connections(self):
-        connections_names = [
-            n for n in self._config.sections() if n.startswith("connections.")
-        ]
-        return {n: self._get_connection(n) for n in connections_names}
+        return self._config["connections"]
```

Comment:
This will return KeyError. In config we have 

```
{
   "connection.a": {},
   "connection.b": {},
   "other_sections": {}
}
```

Reply:
It will not because the config structure is changed